packages/ui/src/components/Button.tsx

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
+import type { Component } from '@react-site-editor/types';
 
-interface Props {
-    text: string;
-    onClick: () => void;
+interface ButtonProps {
```

Comment:
Don't put ComponentProps cause, I want to have the same structure on all components